op-node/rollup/derive/attributes_queue.go

```diff
@@ -43,9 +36,6 @@ func NewAttributesQueue(log log.Logger, cfg *rollup.Config, l1Fetcher L1Receipts
 }
 
 func (aq *AttributesQueue) AddBatch(batch *BatchData) {
-	if aq.progress.Closed {
```

Comment:
Why do some stages panic when adding stuff while closed & others don't?

Reply:
Because that's what the stages expect. This stage can be called with empty batches that are created based on the close. Whereas other stages only ingest data immediately; never ingesting new data when the input is already exhausted.